feat: Add new component - badge

fung-hei-man opened the pull request:
## Description
- Add a new atom component, Badge
- Can be with or without close button

## Github issue

## Related PRs

## Scope
- /

## Screenshot

JiaAnTW commented:
LGTM 👍

JiaAnTW commented on resources/assets/js/components/atom/Badge/style.js:
```diff
@@ -0,0 +1,29 @@
+
+export const BadgeStyle = styled.span`
+    display: flex;
+    flex-direction: row;
```
sorry剛剛沒注意到，這個好像可以不用?

JiaAnTW commented on resources/assets/js/components/atom/Badge/style.js:
```diff
@@ -0,0 +1,29 @@
+    font-size: 14px;
+    border-radius: 30px;
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
```
這三個好像也不需要?

JiaAnTW commented on resources/assets/js/components/atom/Badge/index.js:
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { BadgeStyle, CloseIcon } from './style';
+
+function Badge(props) {
```
props 用解構賦值拆開會比較好維護

fung-hei-man commented:
I was copying the CSS from figma, reviewed & cleared those unused CSS, fixed the style for Chinese (the alignment is different from English...)

Also changed the `props` to params needed.

Please review again, thanks!

JiaAnTW commented:
LGTM 👍
